src/Parser/FrostMage/Modules/Features/IcicleTracker.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class IcicleTracker extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  IciclesWasted = 0;
+
+  on_initialized() {
+	const hasGlacialSpike = this.combatants.selected.hasTalent(SPELLS.GLACIAL_SPIKE_TALENT.id);
+	this.active = hasGlacialSpike;
+  }
+  
+  on_refreshbuff(event) {
+	  if(event.ability.guid === SPELLS.GLACIAL_SPIKE_BUFF.id) {
+		  this.IciclesWasted += 1;
+	  }
+  }
+  
+  suggestions(when) {
+    when(this.IciclesWasted).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> You wasted {this.IciclesWasted} <SpellLink id={SPELLS.ICICLES.id}/>.  Try to cast <SpellLink id={SPELLS.GLACIAL_SPIKE.id}/> once you get to 5 Icicles to avoid wasting them.</span>)
+          .icon(SPELLS.ICICLES.icon)
+          .actual(`${formatNumber(actual)} Icicles Wasted`)
+          .recommended(`${formatNumber(recommended)} is recommended`)
+          .regular(3).major(10);
```

Comment:
It is recommended to use values relative to the `recommended`, so that if you decide to change the values they update as well.

Reply:
I guess i got a little confused about where that recommended number was coming from, so i was having a rough time figuring out how much i needed to adjust recommended since i didnt know the value of recommended. So i just put the numbers instead.

Reply:
The `actual` value is equal to the value in `when()` and `recommended` is the same as the one used in `isLessThan()`, `isGreaterThan()` etc.

Reply:
The recommended is whatever the first number is. Doing them relative isn't required, we don't in the RestoDruid module.